### PR TITLE
Add support for filter pushdown on externalIdPrefix

### PR DESCRIPTION
**Pull request by exyi**

### CODE REVIEW

**Review by emelvarol-cognite — APPROVED** [review]
LGTM 👯

**Comment by emelvarol-cognite** [comment]
version bump

File: src/main/scala/cognite/spark/v1/AssetsRelation.scala [hunk 1]
```diff
@@ -77,7 +78,8 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
       name = m.get("name"),
       source = m.get("source"),
       dataSetIds = m.get("dataSetId").map(idsFromWrappedArray(_).map(CogniteInternalId(_))),
-      labels = m.get("labels").map(externalIdsToContainsAny).getOrElse(None)
+      labels = m.get("labels").map(externalIdsToContainsAny).getOrElse(None),
```
Comment (emelvarol-cognite) [inline src/main/scala/cognite/spark/v1/AssetsRelation.scala:81]: do we need `getOrElse(None)` here? I know it was already like that.

Comment (exyi) [reply src/main/scala/cognite/spark/v1/AssetsRelation.scala:81]: it's the same as flatten in this case - it unwraps one layer of options

Comment (exyi) [reply src/main/scala/cognite/spark/v1/AssetsRelation.scala:81]: Actually, I don't know why didn't we use flatMap, i'll change it

Comment (emelvarol-cognite) [reply src/main/scala/cognite/spark/v1/AssetsRelation.scala]: hehe probably it was me not using scala functions effectively at the time.

**Comment by emelvarol-cognite** [comment]
also changelog

File: src/main/scala/cognite/spark/v1/PushdownUtilities.scala [hunk 2]
```diff
@@ -77,6 +77,8 @@ object PushdownUtilities {
         PushdownFilter("max" + colName.capitalize, value.toString)
       case LessThanOrEqual(colName, value) =>
         PushdownFilter("max" + colName.capitalize, value.toString)
+      case StringStartsWith(colName, value) =>
```
Comment (vuhainguyen) [inline src/main/scala/cognite/spark/v1/PushdownUtilities.scala:80]: I think we should add ` | StringStartsWith(_, null)` into the condition at the line 67 as well?

**Pull request merged**